trieste/acquisition/rule.py

```diff
@@ -2073,6 +2073,183 @@ def update(
         pass
 
 
+class SingleObjectiveTrustRegionDiscrete(UpdatableTrustRegionDiscrete):
+    """
+    An updatable discrete trust region that maintains a set of neighboring points around a
+    single location point. The region is updated based on the best point found in the region.
```

Comment:
we should add some notes on:
- type of discrete variables that should be used here, i.e. discrete numerical
- scaling when it comes to the number of value combinations, i.e. describe the use case

Reply:
I have added some notes. Is that clear?